corehq/form_processor/submission_post.py

```diff
@@ -147,17 +147,16 @@ def _get_success_message(self, instance, cases=None):
 
         Message is formatted with markdown.
         '''
-        default_message = u'   √   '
 
-        if instance.metadata and instance.metadata.deviceID != FORMPLAYER_DEVICE_ID:
-            return default_message
-
-        user = WebUser.get(instance.user_id)
-        if not user:
-            return default_message
+        if not instance.metadata or instance.metadata.deviceID != FORMPLAYER_DEVICE_ID:
+            return u'   √   '
 
         messages = []
-        if instance.metadata.deviceID == FORMPLAYER_DEVICE_ID and user:
+        if instance.metadata.deviceID == FORMPLAYER_DEVICE_ID:
```

Comment:
i think this check isn't needed now that the above check is there

Reply:
updated